.eslintrc.js

```diff
@@ -40,6 +41,8 @@ module.exports = {
         "no-self-assign": "off", // 2 errors
         "no-useless-escape": "off", // 24 errors
         "prefer-spread": "off", // 7 errors
+        "unused-imports/no-unused-imports": "off", // 301 errors
+        "unused-imports/no-unused-vars": "off",
```

Comment:
What does `unused-imports` do for us when these are both off? It looks like these are the only two rules according to https://www.npmjs.com/package/eslint-plugin-unused-imports ?